alpenhorn/db.py

```diff
@@ -87,20 +190,32 @@ def _connect(url=None, db=None):
 class RetryOperationalError(object):
```

Comment:
Not a line changed by you, but the `(object)` isn't need in python 3.
```suggestion
class RetryOperationalError:
```